src/main/kotlin/org/wycliffeassociates/otter/jvm/persistence/repositories/ResourceRepository.kt

```diff
@@ -68,7 +68,7 @@ class ResourceRepository(
                 .subscribeOn(Schedulers.io())
     }
 
-    override fun linkToContent(resource: Content, content: Content): Completable {
+    override fun linkToContent(resource: Content, content: Content, rcFk: Int): Completable {
```

Comment:
I had to add an 'rcFk' parameter to this function and the one below since the Collection object's 'resourceContainer' field is nullable, but rc_fk is not nullable in the Resource table.